sycl/tools/sycl-ls/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+add_executable(sycl-ls sycl-ls.cpp)
+add_dependencies(sycl-ls sycl)
+target_include_directories(sycl-ls PRIVATE "${sycl_inc_dir}")
+target_link_libraries(sycl-ls
+  PRIVATE
+    sycl
+    OpenCL::Headers
```

Comment:
The tool should not depend on OpenCL header and library as it is pure SYCL tool.

Reply:
the OpenCL headers are still the dependency for SYCL headers:
~~~
sycl/include/CL/sycl/detail/common.hpp:17:10: fatal error: CL/cl.h: No such file or directory
 #include <CL/cl.h>
          ^~~~~~~~~
compilation terminated.
~~~

Reply:
From a CUDA backend perspective, it would be good if that dependency is removed, but when we look into it, it was not trivial. This was a while ago,  maybe there is reliance on OpenCL headers now.

Reply:
According to my understanding OpenCL header are included to declare OpenCL interoperability API.